app/src/main/java/com/money/manager/ex/reports/GeneralReportFragment.java

```diff
@@ -66,30 +69,20 @@ private String getHtmlReport(String reportName){
 
         try
         {
-            if(!reportName.trim().isEmpty()) {
-
-                String sql = "SELECT SQLCONTENT FROM REPORT_V1 " +
-                        "WHERE REPORTNAME = '" + reportName + "' " +
-                        "ORDER BY REPORTNAME LIMIT 1";
-
-                Cursor sqlCursor = mDatabase.query(sql, null);
-
-                if(sqlCursor.moveToFirst())
-                {
-                    sqlQuery = sqlCursor.getString(sqlCursor.getColumnIndex("SQLCONTENT"));
-                }
+            ReportRepository repo = new ReportRepository(getActivity());
+            List<Report> report = repo.loadByReportName(reportName.trim());
```

Comment:
`loadByReportName` is redundant, and we can call `repo.loadByName(xxx)` directly here